core/src/stored/reserve.h

```diff
@@ -76,9 +76,9 @@ class ReserveContext {
 
 void InitReservationsLock();
 void TermReservationsLock();
-void _lockReservations(const char* file = "**Unknown**", int line = 0);
+void _lockReservations(const char* = "**Unknown**", int = 0);
 void _unLockReservations();
-void _lockVolumes(const char* file = "**Unknown**", int line = 0);
+void _lockVolumes(const char* = "**Unknown**", int = 0);
 void _unLockVolumes();
```

Comment:
We should remove these parameters as well if these functions do not need them any more.

Reply:
In which chase you can also remove the LockVolumes/LockReservations macros as well and give the functions the correct name.